pyproject.toml

```diff
@@ -31,9 +31,25 @@ classifiers = [
     "Programming Language :: Python :: 3",
 ]
 requires-python = ">=3.8"
+dependencies = [
+    "bashlex",
+]
+
+[project.optional-dependencies]
+dev = [
+    "pytest",
+]
+
+[project.scripts]
+conda-yes = "rapids_pre_commit_hooks.shell.conda_yes:main"
```

Comment:
Do you think this should be named something like `verify-conda-yes`? If I encountered a command `conda-yes` in the wild, I would expect it to invoke `conda --yes` as opposed to being a validator.

Reply:
Renamed to `verify-conda-yes`.